Sprint-2/errors/2.js

```diff
@@ -1,10 +1,13 @@
-
 // Predict and explain first...
 
```

Comment:
Make your prediction FIRST before you run the code, then add an explanation afterwards as well. 

https://github.com/CodeYourFuture/CYF-Workshops/blob/main/asking-questions/readme.md

Reply:
it seems I haven't written down my prediction for some exercises, which was needed to do and was the main purpose of  those exercises to practice Predict,Run,Explain! 
Thank you for reminding me @SallyMcGrath. I'll have a look at all my works and amend them!